pom.xml

```diff
@@ -334,40 +334,40 @@
       <dependency>
         <groupId>org.scala-lang</groupId>
         <artifactId>scala-library</artifactId>
-        <version>2.10.4</version>
+        <version>${scala.version}</version>
       </dependency>
       <dependency>
         <groupId>org.bdgenomics.adam</groupId>
-        <artifactId>adam-core_2.10</artifactId>
+        <artifactId>adam-core_${scala.version.prefix}</artifactId>
```

Comment:
It's OK to use the properties in the `artifactId` for _dependencies_, it doesn't work to use properties in the `artifactId` for the POM itself. I know we've gone back and forth on this a lot, but I think it's actually preferable to do what @ryan-williams is doing here. I'll make a comment at the end with my reasoning.